### DISCUSSION TITLE
How does the service templates work?

### DISCUSSION THREAD

**Discussion by atomicsp** [matt8707/ha-fusion#492] [Q&A] [answered]
I need to show the status from a sensor and make the click do something else. I got this working by adding a custom script in HA. But there is a template here for using built in HA functions such as toggle switch. But for some reason I cant get it to work. The service works just fine in HA dev tools. 
Any help would be appreciated! 
![image](https://github.com/matt8707/ha-fusion/assets/28091120/6bbcb199-15e9-4fff-8ef4-93fd2daa5c69)
![image](https://github.com/matt8707/ha-fusion/assets/28091120/efe573f3-2d2e-4b05-a38f-bf7358263a06)

**Comment by matt8707** [comment, marked as answer]
https://github.com/matt8707/ha-fusion/issues/395#issuecomment-1962990853

**Reply by atomicsp** [reply]
Thanks!

For others, here is what worked:

service: switch.toggle
data:
  entity_id: switch.roborock_revo_q_roborock_vacuum_a75_turned_on
